src/infi/clickhouse_orm/fields.py

```diff
@@ -162,7 +162,25 @@ def validate(self, value):
         if len(value) > self._length:
             raise ValueError('Value of %d bytes is too long for FixedStringField(%d)' % (len(value), self._length))
 
+class BooleanField(Field):
+    # The ClickHouse column type to use
+    db_type = 'UInt8'
```

Comment:
ClickHouse has type `bool` https://clickhouse.com/docs/en/sql-reference/data-types/boolean while it's kept internally as UInt8, why not set here `db_type = 'Bool'` ?

Reply:
DB type has been changed to Bool as suggested. It doesnt make any difference functionally.